Kickstarter-iOS/Views/Controllers/ManageViewPledgeViewController.swift

```diff
@@ -98,6 +132,27 @@ final class ManageViewPledgeViewController: UIViewController {
 
   // MARK: - Configuration
 
+  private func configureChildViewControllers() {
+    let childViewControllers = [
+      self.rewardReceivedViewController
+    ]
+
+    childViewControllers.forEach { viewController in
+      self.addChild(viewController)
+
+      _ = ([viewController.view], self.rootStackView)
+        |> ksr_addArrangedSubviewsToStackView()
+
+      viewController.didMove(toParent: self)
+    }
+  }
+
+  private func setupConstraints() {
+    NSLayoutConstraint.activate([
+      self.rootStackView.widthAnchor.constraint(equalTo: self.rootScrollView.widthAnchor)
```

Comment:
You *usually* need to also set the width of the scroll view to be the equal to the width of the parent VC 🤔

Reply:
Yeah, I know we've done it before but it didn't seem necessary this time ... same on `PledgeVC` 🤔 🐒